packages/server/src/lib/ab.ts

```diff
@@ -31,7 +31,10 @@ export const selectVariant = <V extends Variant, T extends Test<V>>(test: T, mvt
             return control;
         } else {
             const otherVariants = test.variants.filter(v => v.name.toLowerCase() !== 'control');
-            return otherVariants[mvtId % otherVariants.length];
+            if (otherVariants.length > 0) {
```

Comment:
We could potentially do a similar test on line 23 (just to save adding a new `if`) e.g

```ts
    if (test.controlProportionSettings && control && test.variants.length > 1) {
```